### PR TITLE
Vfx/2019.3 staging

### PR DESCRIPTION
**Pull request by julienf-unity**
### Purpose of this PR
VFX backported fixes to 2019.3
- Filter out Direction type on inconsistent operator [Case 1201681](https://issuetracker.unity3d.com/product/unity/issues/guid/1201681/)
- Exclude MouseEvent, RigidBodyCollision, TriggerEvent & Sphere binders when physics modules isn't available
- Visual Effect Activation Track : Handle empty string in ExposedProperty
- in some cases AABox position gizmo would not move when dragged.
- Inspector doesn't trigger any exception if VisualEffectAsset comes from an Asset Bundle [case 1203616](https://issuetracker.unity3d.com/issues/visual-effect-component-is-not-fully-shown-in-the-inspector-if-vfx-is-loaded-from-asset-bundle)
- OnStop Event to the start of a Spawn Context makes it also trigger when OnPlay is sent [Case 1198339] https://issuetracker.unity3d.com/product/unity/issues/guid/1198339/
- Remove unexpected public API : UnityEditor.VFX.VFXSeedMode & IncrementStripIndexOnStart
- Moved VFX Event Tester Window visibility to Component Play Controls SceneView Window

---
### Testing status
**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/vfx%252F2019.3-staging

---
### Comments to reviewers
@iTris666 @PaulDemeulenaere Please double check changes are correct

### CODE REVIEW

File: com.unity.visualeffectgraph/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - New options for quad strips texture mapping: swap UV and custom mapping
 
 ### Fixed
+- Moved VFX Event Tester Window visibility to Component Play Controls SceneView Window
```
Comment (PaulDemeulenaere) [inline com.unity.visualeffectgraph/CHANGELOG.md]: It's strange this entry came at the first place, not really an issue.

Comment (PaulDemeulenaere) [inline com.unity.visualeffectgraph/CHANGELOG.md]: It's strange this entry came at the first place, not really an issue.

File: com.unity.visualeffectgraph/Editor/Utils/VFXResources.cs [hunk 2]
```diff
@@ -48,7 +48,6 @@ private static void Initialize()
 
             if (asset == null)
             {
-                Debug.LogWarning("Could not find " + defaultFileName + ", creating...");
```
Comment (PaulDemeulenaere) [inline com.unity.visualeffectgraph/Editor/Utils/VFXResources.cs]: FYI : This change will allow to merge ono PR https://ono.unity3d.com/unity/unity/pull-request/98914/_/graphics/vfx/feature/compile-on-import-2

**Comment by VladNeykov** [comment]
Unity_CDDt99YlvF

@iTris666 getting errors with a missing `VFXParticleSystemInfo` and `StickyNote/StickyNoteChangeEvent` when I open the latest vfx/2019.3-staging in 2019.3.0f5. Deleted the library folder and am still getting it. Can you please confirm?

**Pull request merged**